### PR TITLE
Style hover

### PR DESCRIPTION
**Pull request by twzkraus**
Added functionality to update styling when user hovers over list or list items

### CODE REVIEW

**Comment by twzkraus** [comment]
@Bouland95

File: src/Styles.jsx [hunk 1]
```diff
@@ -27,11 +27,23 @@ export const RecItemTitle = styled.p`
...
   list-style-type: none;
-  display: inline-block;
+  display: flex;
+  flex-direction: horizontal;
+  justify-content: left;
+  :hover button {
+    visibility: visible;
+    opacity: 1;
+    transition: opacity 0s linear;
+  }
```
Comment (azizbouland) [inline src/Styles.jsx:37]: so you are referencing const NavButton = styled.button at line 91, basically?

**Review by azizbouland — APPROVED** [review]
Well done, I think it's organized and easy to follow. Adding a few comments in List.jsx on why you're checking > 0 or < 3 and passing 1 or -1 on click would improve it.

File: src/Styles.jsx (hunk 1 above)
Comment (twzkraus) [reply src/Styles.jsx:37]: basically. I'm saying, "on hover of RecList, any children who are of type button should have this style." In this case, there are two.

**Comment by twzkraus** [comment]
yeah, thanks for mentioning those. I'm thinking of refactoring to make those more dynamic anyway.

**Pull request merged**